README.md

```diff
@@ -1,6 +1,7 @@
 # RESTEasy
 
-[![Build Status](https://travis-ci.org/resteasy/Resteasy.svg?branch=master)](https://travis-ci.org/resteasy/Resteasy)
+[![Travis Build Status](https://travis-ci.org/resteasy/Resteasy.svg?branch=master)](https://travis-ci.org/resteasy/Resteasy)
```

Comment:
I wonder if we should remove this one as we removed the travis.yml due it going offline tomorrow.

Reply:
@jamezp okay!